exercises/practice/two-fer/src/test/kotlin/TwoFerTest.kt

```diff
@@ -10,21 +10,18 @@ class TwoFerTest {
     }
 
     @Test
-    @Ignore
     fun aNameGiven() {
         assertEquals("One for Alice, one for me.", twofer("Alice"))
     }
 
     @Test
-    @Ignore
     fun anotherNameGiven() {
         assertEquals("One for Bob, one for me.", twofer("Bob"))
     }
 
     @Test
-    @Ignore
     fun emptyStringGiven() {
-        assertEquals("One for , one for me.", twofer(""))
+        assertEquals("One for you, one for me.", twofer(""))
```

Comment:
Good catch, thanks!

Reply:
Let's keep this as it is, without the 'you' if truly we are giving an
empty string as a name.  (The giver of the name is the name expert,
after all, and we asked for it, after all.  Perhaps their parents named
them `""` or they legally changed their name to `""` (good luck with
that, but then Prince…))